repo-agent/pkg/codeserver/codeserver.go

```diff
@@ -0,0 +1,31 @@
+package codeserver
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+var execCommand = exec.Command
+
+func Start() (*exec.Cmd, error) {
+	log.Println("starting code-server")
+	repoURL := os.Getenv("GIT_HTML_URL")
+	parts := strings.Split(strings.TrimPrefix(repoURL, "https://github.com/"), "/")
+	if len(parts) < 4 {
+		return nil, fmt.Errorf("invalid GIT_HTML_URL: %s", repoURL)
+	}
+	repo := parts[1]
+	codeServerPath := "/usr/bin/code-server"
```

Comment:
Hardcoding the path `/workspaces/` makes this function less portable. This path should be passed in as an argument or read from a configuration.